bot/src/slack.js

```diff
@@ -122,10 +122,27 @@ async function loadReplies(channel, ts) {
   return replies
 }
 
+async function catchUpMessages() {
+  const latestReportFrom = (await getLatestReportsByChannel())
+    .reduce((acc, row) => Object.assign(acc, {[row.channel]: row.latest}), {})
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries

Reply:
According to this site: http://kangax.github.io/compat-table/es2016plus/, `Object.fromEntries` is only supported in the latest release of Node (12+). Should I still use it?